packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewYoutube.html

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="iframe-wrapper iframe-wrapper__youtube slds-grid slds-grid_vertical">
+    <div
+      onclick={stopPropagation}
```

Comment:
I don't see the function `stopPropagation` defined in the JS file, we need to define it.